OrbitLinuxTracing/TracerThread.h

```diff
@@ -58,6 +91,20 @@ class TracerThread {
   bool trace_context_switches_ = true;
   bool trace_callstacks_ = true;
   bool trace_instrumented_functions_ = true;
+
+  uint64_t event_count_window_begin_ns_ = 0;
```

Comment:
Nit: Consider creating a small struct / class for stats with Reset / Update methods to cleanly separate them from the main class. (Not sure this is better.)